moulinette/actionsmap.py

```diff
@@ -511,8 +517,13 @@ def generate_cache(klass, namespaces=None):
             with open(am_file, 'r') as f:
                 actionsmaps[n] = yaml.load(f)
 
+            # clean old cached files
+            for i in os.listdir('%s/actionsmap/' % pkg.cachedir):
+                if i.endswith(".pkl"):
```

Comment:
For the principle of testing:

```python
In [10]: %time [with_glob() for x in range(1000000)] and ""
CPU times: user 15.1 s, sys: 3.28 s, total: 18.4 s
Wall time: 18.4 s
Out[10]: ''

In [11]: %time [without_glob() for x in range(1000000)] and ""
CPU times: user 3.55 s, sys: 3.32 s, total: 6.87 s
Wall time: 6.91 s
Out[11]: ''

In [12]: ls
qsd2.pkl  qsd.pkl  ve/
```

Using glob makes it ~ 3 times slower on my computer (no ssd). But yeah, here it really doesn't make any real impact in comparison with everything else we do and I don't think it can be noticeable.